alf/nest/nest.py

```diff
@@ -458,3 +458,33 @@ def _prune(nest, slim_nest):
         raise AssertionError(
             "prune_nest_like() fails between {} and {}".format(
                 nest, slim_nest))
+
+
+def get_field(nested, field):
```

Comment:
There is already a function called "find_field" in this file. What's the difference between them?

Reply:
find_field() finds all the fields with the given name. Here field is a path uniquely identifying the field in the nest.